sql/catalyst/src/main/scala/org/apache/spark/sql/connector/catalog/CatalogV2Util.scala

```diff
@@ -525,10 +525,21 @@ private[sql] object CatalogV2Util {
     }
 
     if (isDefaultColumn) {
-      val e = analyze(f, EXISTS_DEFAULT_COLUMN_METADATA_KEY)
-      assert(e.resolved && e.foldable,
-        "The existence default value must be a simple SQL string that is resolved and foldable, " +
-          "but got: " + f.getExistenceDefaultValue().get)
+      val e = analyze(
+        f,
+        statementType = "",
```

Comment:
Why are we leaving `stetementType` empty? We don't know the statement here, is that why?

Reply:
Yeah, this happens when columns of table are get, usually, that is on analysis, etc, but I don't know on which statements that happens. I saw exception on describe table also. Statement type is used only for message formed for exception, so it was not much important and I left in initially to empty string, but let's put something more descriptive